modules/realvuAnalyticsAdapter.js

```diff
@@ -0,0 +1,975 @@
+      elem.addEventListener(evtType, func, true);
+    } else if (elem.attachEvent) {
+      elem.attachEvent('on' + evtType, func);
+    } else {
+      elem['on' + evtType] = func;
+    }
+  },
+
+  update: function () {
+    var z = this;
+    var de = window.top1.document.documentElement;
+    z.x1 = window.top1.pageXOffset ? window.top1.pageXOffset : de.scrollLeft;
+    z.y1 = window.top1.pageYOffset ? window.top1.pageYOffset : de.scrollTop;
+    var w1 = window.top1.innerWidth ? window.top1.innerWidth : de.clientWidth;
+    var h1 = window.top1.innerHeight ? window.top1.innerHeight : de.clientHeight;
+    z.x2 = z.x1 + w1;
+    z.y2 = z.y1 + h1;
+  },
+  brd: function (s, p) { // return a board Width, s-element, p={Top,Right,Bottom, Left}
+    var u;
+    if (window.getComputedStyle) u = window.getComputedStyle(s, null);
+    else u = s.style;
+    var a = u['border' + p + 'Width'];
+    return parseInt(a.length > 2 ? a.slice(0, -2) : 0);
+  },
+  padd: function (s, p) { // return a board Width, s-element, p={Top,Right,Bottom, Left}
+    var u;
+    if (window.getComputedStyle) u = window.getComputedStyle(s, null);
+    else u = s.style;
+    var a = u['padding' + p];
+    return parseInt(a.length > 2 ? a.slice(0, -2) : 0);
+  },
+  viz_area: function (x1, x2, y1, y2) { // coords of Ad
+    if (this.fr == 1) {
+      try {
+        var iv = Math.round(100 * window.top1.$sf.ext.geom().self.iv);
+        return iv;
+      } catch (e) {
+        /* continue regardless of error */
+      }
+    }
+    var xv1 = Math.max(x1, this.x1);
+    var yv1 = Math.max(y1, this.y1);
+    var xv2 = Math.min(x2, this.x2);
+    var yv2 = Math.min(y2, this.y2);
+    var A = Math.round(100 * ((xv2 - xv1) * (yv2 - yv1)) / ((x2 - x1) * (y2 - y1)));
+    return (A > 0) ? A : 0;
+  },
+
+  viz_dist: function (x1, x2, y1, y2) { // coords of Ad
+    var d = Math.max(0, this.x1 - x2, x1 - this.x2) + Math.max(0, this.y1 - y2, y1 - this.y2);
+    return d;
+  },
+
+  track: function (a, f, params) {
+    var z = this;
+    var s1 = z.tru(a, f) + params;
+    if (f == 'conf') {
+      z.scr(s1, a);
+      z.log(' <a href=\'' + s1 + '\'>' + f + '</a>', a.num);
+    } else {
+      var bk = {
+        s1: s1,
+        a: a,
+        f: f
+      };
+      z.beacons.push(bk);
+    }
+  },
+
+  send_track: function () {
+    var z = this;
+    if (z.sr >= 'a') { // conf, send beacons
+      var bk = z.beacons.shift();
+      while (typeof bk != 'undefined') {
+        bk.s1 = bk.s1.replace(/_sr=0*_/, '_sr=' + z.sr + '_');
+        z.log(' ' + bk.a.riff + ' ' + bk.a.unit_id + /* " "+pin.mode+ */ ' ' + bk.a.w + 'x' + bk.a.h + '@' + bk.a.x + ',' + bk.a.y +
+          ' <a href=\'' + bk.s1 + '\'>' + bk.f + '</a>', bk.a.num);
+        if (bk.a.rnd < Math.pow(10, 1 - (z.sr.charCodeAt(0) & 7))) {
+          z.scr(bk.s1, bk.a);
+        }
+        bk = z.beacons.shift();
+      }
+    }
+  },
+
+  scr: function (s1, a) {
+    var st = document.createElement('script');
```

Comment:
really shouldn't be downloading additional javascript after the module is loaded. What's the use case here? The analytics file is already very large.